lib/linter/.eslintrc.js

```diff
@@ -3,9 +3,6 @@
 module.exports = {
     extends: require.resolve('eslint-config-hapi'),
     parserOptions: {
-        ecmaVersion: 8,
```

Comment:
Thanks for this update. I noticed the same issue and have a PR to fix this file as well https://github.com/hapijs/lab/pull/858